warehouse/upload.go

```diff
@@ -781,6 +783,19 @@ func (job *UploadJob) UpdateTableSchema(tName string, tableSchemaDiff warehouseu
 }
 
 func (job *UploadJob) alterColumnsToWarehouse(tName string, columnsMap model.TableSchema) error {
+	if !job.AlterEnabled {
+		pkgLogger.Debugw("skipping alter columns to warehouse",
+			logfield.SourceID, job.warehouse.Source.ID,
+			logfield.SourceType, job.warehouse.Source.SourceDefinition.Name,
```

Comment:
Shouldn't this be `SourceDefinition.Type`? :thinking:

Reply:
I noticed because you're preceding it with `SourceType`. I just wanted to double check with you in case you would expect some data in the debug log entry but ended up with something else. No need to make changes if this is the expected behaviour.